server/server.go

```diff
@@ -454,20 +454,21 @@ func cutPluginCommandF(w http.ResponseWriter, slashCommand *MMSlashCommand, tag,
 		WriteErrorResponse(w, NewError(err.Error(), nil))
 		return nil
 	}
-
-	msg := fmt.Sprintf("Tag %s created. Waiting for the artifacts to sign and publish.\nWill report back when the process completes.\nGrab :coffee: and a :doughnut: ", tag)
+	
+	cmdFlags := fmt.Sprintf("Flags: tag `%s`, repo `%s`, commitSHA `%s`, force `%v`.", tag, repo, commitSHA, force)
+	msg := fmt.Sprintf("%s\nTag created. Waiting for the artifacts to sign and publish.\nWill report back when the process completes.\nGrab :coffee: and a :doughnut: ", cmdFlags)
 	if err := createTag(ctx, client, Cfg.GithubOrg, repo, tag, commitSHA); errors.Is(err, ErrTagExists) {
 		if !force {
-			WriteErrorResponse(w, NewError(fmt.Errorf("Tag %s already exists, not generating any artifacts. Use --force to regenerate artifacts.", tag).Error(), nil))
+			WriteErrorResponse(w, NewError(fmt.Errorf("%s\nTag already exists. Not generating any artifacts. Use --force to regenerate artifacts. ", cmdFlags).Error(), nil))
```

Comment:
Since the initial msg contains all the cmd flags maybe lets not add it here 

```suggestion
			WriteErrorResponse(w, NewError(fmt.Errorf("Tag already exists. Not generating any artifacts. Use --force to regenerate artifacts. ", cmdFlags).Error(), nil))
```

Reply:
```suggestion
	WriteErrorResponse(w, NewError(fmt.Errorf("Tag %s already exists. Not generating any artifacts. Use --force to regenerate artifacts.", tag).Error(), nil))
```